python/GafferSceneUI/TransformToolUI.py

```diff
@@ -101,6 +109,21 @@ def _distance( ancestor, descendant ) :
 
 	return result
 
+class _TargetTipWidget( GafferUI.Frame ) :
+
+	def __init__( self, tool, **kw ) :
+
+		GafferUI.Frame.__init__( self, borderWidth = 4, **kw )
+
+		label = Gaffer.Metadata.value( tool, "ui:transformTool:toolTips" ) or ""
```

Comment:
Maybe this should be "ui:transformTool:toolTip" seeing as it's now a single string?

Reply:
Yes, that should be singular now, fixed and squashed into https://github.com/GafferHQ/gaffer/pull/5558/commits/f22a800cf4f8c291440786a844ec7f399322180e